pyproject.toml

```diff
@@ -174,6 +174,9 @@ warn_untyped_fields = true
 [tool.pydocstyle]
 convention = "google"
 
+[tool.darglint]
+docstring_style = "google"
```

Comment:
Explicitly calling this out for clarity, even though it is the default.

Reply:
Thanks for this 🙇🏼 Helps slow people like me :)